rakelimit.go

```diff
@@ -24,7 +25,7 @@ func NewRakelimit(ppsLimit float64) (*Rakelimit, error) {
 	// set ratelimit
 	collectionSpec := rakelimitSpec.CollectionSpec()
 	if err := collectionSpec.RewriteConstants(map[string]interface{}{
-		"limit": floatToFixed(ppsLimit),
+		"limit": floatToFixed(float64(ppsLimit)),
```

Comment:
Nit: we don't need to go via float anymore, I think. You can turn `limit` in the BPF into a `__u32` and do the conversion in BPF.